### PR TITLE
Client side mixpanel tracking

### PR DESCRIPTION
**Pull request by petesantos**
This is the check in of the code we mobbed on to add tracking to the share a workspace event.
This code can be used to add additional captures to the system as well as enhance the capture with additional metadata (e.g. time, app id)

### CODE REVIEW

File: config/dev.json [hunk 1]
```diff
@@ -19,5 +19,6 @@
     "appId": "saturnnonprod-O9lUP",
     "apiKey": "AQEBBAEkx4iE2KxNyI7Wx08EwU1ycTM7E4FMSmaibbMUQxNU6uQvuAJt7fyABAtFYSYfgEE"
   },
-  "tosUrlRoot": "https://us-central1-broad-workbench-tos-dev.cloudfunctions.net/tos"
+  "tosUrlRoot": "https://us-central1-broad-workbench-tos-dev.cloudfunctions.net/tos",
+  "metricsRoot": "https://terra-metrics-dev.appspot.com/"
```
Comment (zarsky-broad) [inline config/dev.json]: nit: alphabetize, also remove slash

File: src/libs/ajax.js [hunk 2]
```diff
@@ -109,6 +111,7 @@ const fetchOrchestration = _.flow(withUrlPrefix(`${getConfig().orchestrationUrlR
 const fetchRex = withUrlPrefix(`${getConfig().rexUrlRoot}/api/`, fetchOk)
 const fetchBond = withUrlPrefix(`${getConfig().bondUrlRoot}/`, fetchOk)
 const fetchMartha = withUrlPrefix(`${getConfig().marthaUrlRoot}/`, fetchOk)
+const fetchMetrics = withUrlPrefix(`${getConfig()}.metricsRoot`, fetchOk)
```
Comment (zarsky-broad) [inline src/libs/ajax.js]:
```suggestion
const fetchMetrics = withUrlPrefix(`${getConfig().metricsRoot}/`, fetchOk)
```

File: src/libs/ajax.js [hunk 3]
```diff
@@ -873,7 +876,6 @@ const GoogleBilling = signal => ({
   }
 })
 
-
```
Comment (zarsky-broad) [inline src/libs/ajax.js]: gimme back my line

**Review by zarsky-broad — APPROVED** [review]

File: config/dev.json [hunk 4]
```diff
@@ -19,5 +20,5 @@
     "appId": "saturnnonprod-O9lUP",
     "apiKey": "AQEBBAEkx4iE2KxNyI7Wx08EwU1ycTM7E4FMSmaibbMUQxNU6uQvuAJt7fyABAtFYSYfgEE"
   },
-  "tosUrlRoot": "https://us-central1-broad-workbench-tos-dev.cloudfunctions.net/tos"
+  "tosUrlRoot": "https://us-central1-broad-workbench-tos-dev.cloudfunctions.net/tos",
```
Comment (zarsky-broad) [inline config/dev.json]:
```suggestion
  "tosUrlRoot": "https://us-central1-broad-workbench-tos-dev.cloudfunctions.net/tos"
```

File: src/libs/ajax.js (hunk 3 above)
Comment (zarsky-broad) [reply src/libs/ajax.js]: gimme back my 2 lines

Comment (petesantos) [reply src/libs/ajax.js]: sorry - misinterpreted this the other way

**Pull request merged**